tests/conftest.py

```diff
@@ -79,3 +80,22 @@ def _import_execute(source: str, *, custom_module_name: str = None):
             print('KeyboardInterrupt')
 
     return _import_execute
+
+
+if sys.platform == 'emscripten':
```

Comment:
```suggestion
if sys.platform == 'emscripten':
    # we have to stub pytest-benchmark since it can't currently be used with webassembly/emscripten
```